pkgs/misc/emulators/retroarch/hashes.json

```diff
@@ -352,22 +380,22 @@
     "picodrive": {
         "owner": "libretro",
         "repo": "picodrive",
-        "rev": "9cb99ce36f93871b05c5adc2790b2e33e63b50b6",
-        "sha256": "hhdEuri1hSNLAkno8Rwd0Yls11Yh6Q7/+t4T2LH+BaE=",
+        "rev": "3edf1a00f64e0f22331233bb1638170115b2ac2e",
+        "sha256": "4IWYOJ2wTDkdO4FxsAWCV724VNViHIb42nYc+j4pekU=",
         "fetchSubmodules": true
     },
     "play": {
         "owner": "jpd002",
         "repo": "Play-",
-        "rev": "b8e16159734c2068db0f2f12b11bc16ef55058ce",
-        "sha256": "qjp1rEjmDAAB2wXITA3lAS+ERJuZinoneJToYiYRi/w=",
+        "rev": "6b9cc418004c01a195c78387752cc99245ba54d5",
+        "sha256": "sha256-7nU5fQ8pQLmKy9Swmshkv2oj+HV0oTcqQ93LfNSq1us=",
```

Comment:
Had to update the hash manually here, since the last update has some git module issues that `nix-prefetch-github` is having difficult to parse. This is why the hash here is in the new format (I am using `nix 2.4`).